acoustid/future/worker.py

```
@@ -0,0 +1,79 @@
+logger = logging.getLogger("aserver.worker")
+
+
+async def handle_message(msg: Msg) -> None:
+    print(msg.headers)
+    print(msg.metadata)
+    await msg.ack()
+
+
+async def worker(
+    nats_servers: list[str],
+    postgres_app_dsn: str,
```

Ruff (0.8.2)

Unused function argument: `postgres_app_dsn`

(ARG001)